deel/influenciae/rps/rps_lje.py

```diff
@@ -52,81 +47,112 @@ def __init__(
             ihvp_calculator_factory: InverseHessianVectorProductFactory,
             n_samples_for_hessian: Optional[int] = None,
             target_layer: Union[int, str] = -1,
-            shuffle_buffer_size: int = 10000
+            shuffle_buffer_size: int = 10000,
+            epsilon: float = 1e-5
     ):
-        # Use a FirstOrderInfluenceCalculator to compute the jacobian expanded weights for the model
-        ihvp_calculator = ihvp_calculator_factory.build(influence_model, dataset)
-        first_order_calculator = FirstOrderInfluenceCalculator(model=influence_model,
-                                                               dataset=dataset,
-                                                               ihvp_calculator=ihvp_calculator,
-                                                               n_samples_for_hessian=n_samples_for_hessian,
-                                                               shuffle_buffer_size=shuffle_buffer_size,
-                                                               normalize=False)
-        influence_vector_dataset = first_order_calculator.compute_influence_vector(dataset)
-
-        # Compute weight factor for the optimization step
-        size = tf.data.experimental.cardinality(dataset)
-        iter_dataset = iter(influence_vector_dataset)
-        weight_size = tf.reduce_sum(
-            tf.stack([tf.reduce_prod(tf.shape(w)) for w in influence_model.model.layers[target_layer].weights])
+        super().__init__(influence_model.model, dataset, influence_model.loss_function)
+        self.epsilon = tf.constant(epsilon, dtype=tf.float32)
+
+        # In the paper, the authors explain that in practice, they use a single step of SGD to compute the
```

Comment:
Later, you say you accumulate the gradient, so i guess it is a single GD step no?

Reply:
Yup, it's a single gradient step on the amount of samples specified by `n_samples_for_hessian`.

Reply:
Then, should not we change the comment or do you think it's overkill?